synadm/cli/__init__.py

```diff
@@ -300,10 +314,10 @@ def generate_mxid(self, user_id):
     """)
 @click.option(
     "--output", "-o", default="",
-    type=click.Choice(["yaml", "json", "human", "pprint",
-                       "y", "j", "h", "p", ""]),
+    type=click.Choice(["yaml", "json", "minified", "human", "pprint",
+                       "y", "j", "m", "h", "p", ""]),
```

Comment:
Why is an empty string an option?

Reply:
I was aksing that myself when I went over that code and recall something very very dusty in my brain that there once was a reason. It might be no reason anymore. It needs to be tested thoroughly whether or not in can be removed...